feat:[PIPE-19105]: Add image for pipeline ended and success for notification (#6756)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/cd-steps/notify-users-of-pipeline-events.md

Please note that you should be able to include custom headers while creating custom webhook notifications channel(s) as well as while updating the existing ones.
:::

> For an example, if you need to send slack notification when pipeline completes as well as when it is successful it will look something like:-

Reviewer: ```suggestion
For example, Slack notifications when the pipeline completes as well as when it is successful looks something like this:
```

Reviewer: Always use : instead of :-
